doc/using-logging.rst

```diff
@@ -0,0 +1,117 @@
+=====================
+ Configuring Logging
+=====================
+
+One common use of ZConfig is to configure the Python :mod:`logging`
+framework. ZConfig provides one simple convenience function to do this:
+
+.. autofunction:: ZConfig.configureLoggers
+
+
+This function takes the configuration text::
+
+    >>> from ZConfig import configureLoggers
+    >>> configureLoggers('''
```

Comment:
Ok, I think I get it now. Working on it.

(I think I'll still avoid manuel for now because there are several tests that use doctest directly and I don't want to convert them all right now. That can be a followup IMO.)